rules/integrations/aws/initial_access_iam_session_token_used_from_multiple_addresses.toml

```diff
@@ -1,37 +1,41 @@
 """
 false_positives = [
     """
-    Highly distributed environments (e.g., globally deployed automation or edge nodes) may cause a single IAM user to
-    appear from multiple IPs. Review the geolocation and automation context to rule out benign use.
+    Highly distributed environments (e.g., globally deployed automation or edge nodes) may cause a single IAM user to appear from multiple IPs. 
+    Review the geolocation, network context, and user agent patterns to rule out benign use.
     """,
 ]
-from = "now-30m"
+from = "now-32m"
+interval = "5m"
 language = "esql"
 license = "Elastic License v2"
-name = "AWS STS Temporary IAM Session Token Used from Multiple Addresses"
+name = "AWS Access Token Used from Multiple Addresses"
 note = """## Triage and Analysis
 
-### Investigating AWS STS Temporary IAM Session Token Used from Multiple Addresses
+### Investigating AWS Access Token Used from Multiple Addresses
 
-Temporary session tokens (typically starting with 'ASIA') are expected to be short-lived and bound to a single user session. Usage from multiple IP addresses may indicate the token was stolen and used elsewhere.
+Access tokens are bound to a single user. Usage from multiple IP addresses may indicate the token was stolen and used elsewhere. By correlating this with additional detection criteria like multiple user agents, different cities, and different networks, we can improve the fidelity of the rule and help to eliminate false positives associated with expected behavior, like dual-stack IPV4/IPV6 usage. 
 
 #### Possible Investigation Steps
 
-- **Identify the IAM User**: Examine `aws.cloudtrail.user_identity.arn` and correlate with `source.ip` to determine how widely the token was used.
+- **Identify the IAM User**: Examine the `aws.cloudtrail.user_identity.arn` stored in `user_id` and correlate with the `source.ips` stored in `ip_list` and `unique_ips` count to determine how widely the token was used.
+- **Correlate Additional Detection Context**: Examine `activity_type` and `fidelity_score` to determine additional cities, networks or user agents associated with the token usage. 
 - **Check Recent MFA Events**: Determine whether the user recently enabled MFA, registered devices, or assumed a role using this token.
-- **Review Workload Context**: Confirm whether the user was expected to be active in multiple regions or environments.
+- **Review Workload Context**: Confirm whether the user was expected to be active across multiple cities, networks or user agent environments.
 - **Trace Adversary Movement**: Pivot to related actions (e.g., `s3:ListBuckets`, `iam:ListUsers`, `sts:GetCallerIdentity`) to track further enumeration.
```

Comment:
Maybe just add an investigation step to check whether its short-term (ASIA...) or long-term (AKIA...)?